Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:slim
```

Comment:
is this image with python 3.10?

Reply:
The image used will be the latest stable python version, so 3.12.2 right now. I don't see a problem with this _yet_, but when the need arises, we can pin the last working version.